README.md

```diff
@@ -106,10 +105,12 @@ This implementation also supports **tracking changes in [linked content](https:/
 
     ```
     {
```

Comment:
Consider linking the actual type from the readme.

Reply:
added permalinks, unfortunately, GH cannot display previews of links to a file on a branch so the links are to a specific commit. Because of this, the readme needs to be updated on every change to the configuration (even a rename) which would not be necessary for a branch-link.